setup.py

```diff
@@ -55,7 +56,6 @@
     python_requires=">=3.6",
     setup_requires=["cython", "numpy"],
     install_requires=REQUIRES,
-    include_dirs=[numpy.get_include()],
```

Comment:
I didn't understand this option. At least, I couldn't find the description about this on the distutils' docs.
https://docs.python.org/3/distutils/setupscript.html#preprocessor-options